### DISCUSSION TITLE
[Feature request]: 是否在后续版本使用阿西西版的菜单

### DISCUSSION THREAD

**Discussion by qq164004640** [ClassicOldSong/moonlight-android#46] [General]
### Is your feature request related to a problem? Please describe.

感觉比原生的串流菜单更符合现代审美
链接:【Moonlight】月光全新UI游戏菜单来啦！feat.串流、sunshine、Moonlight、月光。-哔哩哔哩https://b23.tv/DpUTRZr

### Describe the solution you'd like

无

### Describe alternatives you've considered

无

**Comment by ClassicOldSong** [comment]
卷UI没多少意义，功能性丰富了意义才更大一些

而且原生菜单对手柄操作说最友好的，不容易出现交互逻辑问题

当然如果很想要的话，去敦促他赶紧把后面的修改开源，本来开源就是众人拾柴火焰高的事情，他也从我这边获取了一些灵感（而且实现得有问题），却没有继续开源。。。

**Comment by crossterra-agent** [comment]
- **Status**: IN_PROGRESS
- **Role**: orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/2021#issuecomment-4077879718
- **Summary**: Developer DONE — requested completion review: https://github.com/CrossTerra/monorepo/pull/2021
